CHANGELOG.md

```diff
@@ -52,6 +52,7 @@ Ref: https://keepachangelog.com/en/1.0.0/
 
 ### Bug Fixes
 
+* (x/auth) [#18564](https://github.com/cosmos/cosmos-sdk/pull/18564) Fix total fees calculation when batch signing.
```

Comment:
Note for myself, search in the whole main repo for issue number when changelog merging after tag of a patch release, given that a changelog can be under all submodules from now on main.